doc/source/quickstart.rst

```diff
@@ -70,25 +40,49 @@ scripts and XML files to configure a case (see below) for the requested
 resolution, component set, and machine. **create_newcase** has several
 required arguments (invoke **create_newcase --help** for help).
 
-If running on a supported machine, (``$MACHINE``), that machine will
+The ``$PROJECT`` variable must be specified in either your 
+shell environment or as the ``--project`` argument to **create_newcase**.
+
+If running on a supported machine, that machine will
 normally be recognized automatically and therefore it is not required
 to specify the ``--machine`` argument to **create_newcase**, then
 invoke **create_newcase** as follows:
 
 .. code-block:: console
 
-    create_newcase --case $CASEROOT --compset $COMPSET --res $GRID 
+    ./create_newcase --case $CASEROOT --compset $COMPSET --res $GRID
+
+An example on NCAR machine cheyenne:
+
+.. code-block:: console
+
+    ./create_newcase --case /glade/scratch/[username]/cases/b.e20.B1850.f19_g17.test --compset B1850 --res f19_g17 
+
+where:
+
+-  ``$COMPSET`` refers to the `component set <http://www.cesm.ucar.edu/models/cesm2.0/config/compsets.html>`_.
+
+-  ``$GRID`` refers to the model `resolution <http://www.cesm.ucar.edu/models/cesm2.0/config/grids.html>`_.
+
+-  ``$CASEROOT`` refers to the full pathname of the root directory where the
+   case (``$CASE``) will be created. In the example above, the ``$CASEROOT = /glade/scratch/[username]/cases/A.f19_g17.test``
+   where [username] is your cheyenne login name.
+
+-  ``$CASE`` refers to the case name, which is the basename of $CASEROOT.
+   See `CESM2 Experiment Casenames  <http://www.cesm.ucar.edu/models/cesm2.0/naming_conventions.html#casenames>`_
+   for details regarding CESM experiment case naming conventions.  In the example above, ``$CASE = A.f19_g17.test``.
+
+The ``--run-unsupported`` argument force the creation of a case that is not tested or supported by CESM developers. 
+
 
-If running on a new target machine, see
-`the CIME porting guide <http://esmci.github.io/cime/users_guide/porting-cime.html>`_.
 
 Setting up the case run script
 ==============================
 
 Issuing the `case.setup`_ command creates a ``$CASEROOT/case.run`` script
```

Comment:
In recent versions of cime, the script is hidden: ``.case.run`` rather than ``case.run``. The point of hiding it is so that users aren't tempted to muck with it themselves, I think, so maybe that shouldn't be documented explicitly. This could just say something more general, like "Issuing the ``case.setup`` command creates scripts needed to run the model along with namelist...